controllers/storagecluster/generate.go

```diff
@@ -136,14 +136,32 @@ func generateNameForCephRbdMirror(initData *ocsv1.StorageCluster) string {
 
 // generateCephReplicatedSpec returns the ReplicatedSpec for the cephCluster
 // based on the StorageCluster configuration
-func generateCephReplicatedSpec(initData *ocsv1.StorageCluster, poolType string) cephv1.ReplicatedSpec {
+func generateCephReplicatedSpec(initData *ocsv1.StorageCluster, poolType string, storageType string) cephv1.ReplicatedSpec {
 	crs := cephv1.ReplicatedSpec{}
 
 	crs.Size = getCephPoolReplicatedSize(initData)
 	crs.ReplicasPerFailureDomain = uint(getReplicasPerFailureDomain(initData))
-	//lint:ignore ST1017 required to compare it directly
-	if "data" == poolType {
-		crs.TargetSizeRatio = .49
+	if poolType == "data" {
```

Comment:
Can you pls create const variables for `block`, `file` etc and use them at all places?

Reply:
Changed